test/test_utils.py

```diff
@@ -633,6 +635,16 @@ class GenericObject:
 
         assert len(get_type_hints(GenericObject())) == 0
 
+    def test_is_special_typed_parameter(self):
+        def func(a: List[str], b: Literal["a", "b"], c, d: Request):
+            pass
+
+        hints = get_type_hints(func)
+        assert not is_special_typed_parameter("a", hints)
+        assert not is_special_typed_parameter("a", hints)
+        assert not is_special_typed_parameter("a", hints)
+        assert not is_special_typed_parameter("a", hints)
```

Comment:
I believe this was intended? 
```suggestion
        assert not is_special_typed_parameter("a", hints)
        assert not is_special_typed_parameter("b", hints)
        assert not is_special_typed_parameter("c", hints)
        assert is_special_typed_parameter("d", hints)
```

Reply:
Lol yes 🙈